feat: create official Netlify Remix template (#83)
netlify/remix-template: remix.init/README-edge.md

# Welcome to Remix!

- [Remix Docs](https://remix.run/docs)
- [Netlify Edge Functions Overview](https://docs.netlify.com/netlify-labs/experimental-features/edge-functions)

## Netlify Setup

> 1. Install the [Netlify CLI](https://www.netlify.com/products/dev/):

Reviewer: Same as the other README, if you want you can link straight to the docs:

```suggestion
1. Install the [Netlify CLI](https://docs.netlify.com/cli/get-started/):
```

> Open up [http://localhost:3000](http://localhost:3000), and you should be ready to go!

Author: I've updated the Netlify Edge Functions and Netlify READMEs to be exactly the same aside from the link to Netlify Functions or Edge Functions docs. That should be the last changes required @stephmarie17 unless you spot anything else. Thanks again!

Note: When running the Netlify CLI, file changes will rebuild assets, but you will not see the changes to the page you are on unless you do a browser refresh of the page. Due to how the Netlify CLI builds the Remix App Server, it does not support hot module reloading.

> ### Excluding routes

Author: This is a new section I added @stephmarie17. Could the wording be improved?

Reviewer: Left a suggestion below!

> If you want to exclude routes for non-Remix code, e.g. for custom Netlify Functions or Netlify Edge Functions, add an additional entry in the array, e.g.:

Reviewer: ```suggestion
You can exclude routes for non-Remix code such as custom Netlify Functions or Edge Functions. To do this, add an additional entry in the array like in the example below:
```
